v3-sdk/swap-and-add-liquidity/src/libs/liquidity.ts

```diff
@@ -194,7 +212,7 @@ export async function getTokenTransferApproval(
     )
 
     const transaction = await tokenContract.populateTransaction.approve(
-      NONFUNGIBLE_POSITION_MANAGER_CONTRACT_ADDRESS,
+      spenderAddress,
```

Comment:
Should we put this used contract into the example's function instead of function input? I don't mind the argument here, but maybe we need to go back and make this consistent with others then too?

Reply:
via slack, needed for this case!